### ISSUE TITLE
[Bug]: Instant crash on Windows 7

### ISSUE THREAD

**Issue by ArmanHayots** [eranif/codelite#3076]
### What happened?

Windows 7 x64 SP1.
Version 16.0 from site.
Closes just after opening.
Eventlog attached.

### Version

16.0.0

### Operating system

Other

### Steps to reproduce

```text
Download and install using admin rights. Try to open.
```

### Relevant log output

```shell
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Application Error" /> 
  <EventID Qualifiers="0">1000</EventID> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2022-09-16T08:24:34.000000000Z" /> 
  <EventRecordID>35561</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>SERVER-ARMAN</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>codelite.exe</Data> 
  <Data>16.0.0.0</Data> 
  <Data>625cef3f</Data> 
  <Data>ntdll.dll</Data> 
  <Data>6.1.7601.24548</Data> 
  <Data>5e3a68b8</Data> 
  <Data>c0000374</Data> 
  <Data>00000000000bf272</Data> 
  <Data>1808</Data> 
  <Data>01d8c9a5bb105983</Data> 
  <Data>C:\Program Files\CodeLite\codelite.exe</Data> 
  <Data>C:\Windows\SYSTEM32\ntdll.dll</Data> 
  <Data>fe043e55-3598-11ed-ab65-68b599528092</Data> 
  </EventData>
  </Event>
```

**Comment by eranif** [comment]
I do not have Windows 7 so I can't try it. The Windows installer is for Windows 10 and later
I can only suggest to try and install CodeLite 16.4.0 (released last week)

It was built with a different toolchain that might also work properly on Winodws 7

**Comment by eranif** [comment]
fixed by: 41ba11a43
